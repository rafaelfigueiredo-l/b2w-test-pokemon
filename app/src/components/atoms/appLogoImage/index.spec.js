import AppLogoImage from ".";

describe("<AppLogoImage />", () => {
  const image = require("../../../assets/images/pokemonImageNotFound.png");

  const component = <AppLogoImage alt={"AltText"} src={image} />;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly AppLogoImage component", () => {
    expect(wrapper.find(".appLogoImage")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly AppLogoImage component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should render correctly alt property", () => {
    expect(wrapper.prop("alt")).toEqual("AltText");
  });

  it("should render correctly alt property", () => {
    expect(wrapper.prop("src")).toEqual(image);
  });
});
