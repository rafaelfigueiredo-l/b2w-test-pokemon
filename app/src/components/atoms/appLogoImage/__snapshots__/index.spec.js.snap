// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLogoImage /> should mount correctly AppLogoImage component 1`] = `
<styled.img
  alt="AltText"
  className="appLogoImage"
  src="pokemonImageNotFound.png"
>
  <img
    alt="AltText"
    className="sc-AxjAm dPerqe appLogoImage"
    src="pokemonImageNotFound.png"
  />
</styled.img>
`;
